src/notes.ts

```diff
@@ -208,19 +219,19 @@ chrome.storage.onChanged.addListener((changes, areaName) => {
         }
       }
     }
+
+    if (changes["clipboard"]) {
+      state.clipboard = changes["clipboard"].newValue;
+    }
   }
 
   if (areaName === "sync") {
     if (changes["selection"]) {
       const selection = changes["selection"].newValue as ContextMenuSelection;
-      if (!selection) { return; }
+      if (!selection || !selection.text) { return; }
       chrome.storage.local.get(["id"], local => {
         if (selection.sender === local.id) { return; }
-        const notes: NotesObject = { ...state.notes };
-        if ("Clipboard" in notes) {
-          notes["Clipboard"].content = selection.text + notes["Clipboard"].content;
-          chrome.storage.local.set({ notes: notes });
-        }
+        sendMessage(MessageType.SAVE_TO_CLIPBOARD, selection.text);
```

Comment:
Saving selected text to a local computer, or to other computer(s), shares the same underlying steps and can be now done by simply sending a message.